api/queries_pr.go

```diff
@@ -264,6 +265,7 @@ func PullRequests(client *Client, repo ghrepo.Interface, currentPRNumber int, cu
 		state
 		url
 		headRefName
+		mergeable
```

Comment:
Is it necessary to request this field if we're never going to consume it when listing pull requests? This is a potentially expensive field as GitHub [starts a background job](https://developer.github.com/v3/git/#checking-mergeability-of-pull-requests) to check mergeability.